### DISCUSSION TITLE
SelfControl not working on BigSur version 11.3

### DISCUSSION THREAD

**Discussion by jeanweb90** [SelfControlApp/selfcontrol#716] [Q&A] [answered]
App opens and UI works perfectly but site blocking does absolutely nothing.. Anyone else having problems? How to fix?

**Comment by cstigler** [comment, marked as answer]
Hi @jeanweb90 , try downloading the newest version of SelfControl released today (version 4.0) from our website: https://selfcontrolapp.com/

Then restart your computer and see if blocking works then. It's working for me and others on Big Sur so likely something specific to your system!
